servers/src/test/java/tachyon/master/file/FileSystemMasterTest.java

```diff
@@ -121,10 +124,12 @@ public void getNewBlockIdForFileTest() throws Exception {
 
   @Test
   public void createFileWithTTLTest() throws Exception {
-    long fileId = mFileSystemMaster.create(NESTED_FILE_URI, Constants.KB, true, 1);
+    long fileId = mFileSystemMaster.create(NESTED_FILE_URI, Constants.KB, true, 0);
     FileInfo fileInfo = mFileSystemMaster.getFileInfo(fileId);
     Assert.assertEquals(fileInfo.fileId, fileId);
-    CommonUtils.sleepMs(5000);
+    HeartbeatScheduler.await(HeartbeatContext.MASTER_TTL_CHECK);
```

Comment:
Shall we have a timeout mechanism in HeartbeatScheduler, so that we know for sure the tests won't hang forever?